tests/FatBinaryBufferTest.php

```diff
@@ -513,7 +516,7 @@ public function testMixedWithEndian()
         $binaryBufferLE->writeUChar(233);
         $binaryBufferLE->writeStringByLength("gogogo", 3);
         
-        $newBuffer = new FatBinaryBuffer((mt_rand(1, 10) % 2) === 0);
+        $newBuffer = new FatBinaryBuffer($isBigEndian);
         $newBuffer->setBuffer($binaryBufferLE->getBuffer());
```

Comment:
`$newBuffer` should keep rand endian, and use `$newBuffer->readString($isBigEndian)` afterwards.

Reply:
Yes, you are rigth. I fix it.